docs/excel/show-hide-add-in.md

```diff
@@ -88,9 +86,11 @@ var removeVisibilityModeHandler =
 removeVisibilityModeHandler();
 ```
 
-The `onVisibilityModeChanged` method is asynchronous which means that if your code calls the *deregister* handler that `onVisibilityModeChanged` returns, you should ensure that `onVisibilityModeChanged` has completed before calling the deregister handler. One way to do that is to use the `await` keyword on the method call as in the following example.
+The `onVisibilityModeChanged` method is asynchronous and returns a promise, which means that your code needs to await the promise before it can call the **deregister** handler.
```

Comment:
```suggestion
The `onVisibilityModeChanged` method is asynchronous and returns a promise, which means that your code needs to await the fulfillment of the promise before it can call the **deregister** handler.
```